<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion De Stock 1.0.1 Par Senisna B.D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon1.ico</normaloff>icon1.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 103, 217);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;ManaraDocs Amatti Font&quot;;
background-color: rgb(69, 173, 173);
color: rgb(70, 106, 199);
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;تسيير مخزون المؤسسات التربوية&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(0, 255, 255);
font: 14pt &quot;ManaraDocs Amatti Font&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>تفصيل المخزون</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;البحث عن التعيين الذي مخزونه يساوي أو اقل من  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>731</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الرمز</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>التعيين</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الدخول</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الخروج</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>المخزون</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>المستفيد</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الممون</string>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="stock_filter_txt">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(99, 107, 255);</string>
      </property>
      <property name="text">
       <string>بحث</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>إحصاء المخزون</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>16</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>عدد المواد حسب الرمز</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_designation_nbr">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(58, 176, 189);
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; المواد  من حيث المخزون المتبقي ( من الأصغر للأكبر)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>431</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(58, 176, 189);
font: 11pt &quot;ManaraDocs Amatti Font&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>المخزون</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>التعيين</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>الرمز</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="check_btn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>220</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(61, 88, 206);
color: rgb(0, 0, 0);
background-color: rgb(58, 176, 189);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ِCheck</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>تسيير المخزون</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-30</y>
        <width>801</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(0, 255, 255);
font: 14pt &quot;ManaraDocs Amatti Font&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>تفصيل المخزون</string>
       </attribute>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>10</y>
          <width>371</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;البحث عن التعيين الذي مخزونه يساوي أو اقل من  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="table_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>731</width>
          <height>341</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>الرمز</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>التعيين</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>الدخول</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>الخروج</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>المخزون</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>المستفيد</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>الممون</string>
         </property>
        </column>
       </widget>
       <widget class="QSpinBox" name="stock_filter_txt_2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>10</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="search_btn_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(99, 107, 255);</string>
        </property>
        <property name="text">
         <string>بحث</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>إحصاء المخزون</string>
       </attribute>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>16</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>عدد المواد حسب الرمز</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_designation_nbr_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>20</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; المواد  من حيث المخزون المتبقي ( من الأصغر للأكبر)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="table2_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>431</width>
          <height>351</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
font: 11pt &quot;ManaraDocs Amatti Font&quot;;</string>
        </property>
        <column>
         <property name="text">
          <string>المخزون</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>التعيين</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>الرمز</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="check_btn_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>220</y>
          <width>121</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(61, 88, 206);
color: rgb(0, 0, 0);
background-color: rgb(58, 176, 189);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>ِCheck</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>تسيير المخزون</string>
       </attribute>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>40</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>230</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>المخزون</string>
        </property>
       </widget>
       <widget class="QLabel" name="beneficiare">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>190</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="fournisseur">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>150</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="code">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="designation">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>110</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>190</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>المستفيذ</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>110</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>التعيين</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>150</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>المصدر(الممون)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>الرمز</string>
        </property>
       </widget>
       <widget class="QLabel" name="id">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>40</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="last_btn">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>280</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
font: 20pt &quot;Stencil&quot;;
color: rgb(241, 255, 76);</string>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="next_btn">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>280</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
font: 20pt &quot;Stencil&quot;;
color: rgb(241, 255, 76);</string>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="first_btn">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>280</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
font: 20pt &quot;Stencil&quot;;
color: rgb(241, 255, 76);</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="previous_btn">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>280</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(58, 176, 189);
font: 20pt &quot;Stencil&quot;;
color: rgb(241, 255, 76);</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_btn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>280</y>
          <width>81</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(78, 166, 189);
</string>
        </property>
        <property name="text">
         <string>إضافة</string>
        </property>
       </widget>
       <widget class="QPushButton" name="update_btn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>280</y>
          <width>81</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(78, 166, 189);
color: rgb(72, 255, 52);
</string>
        </property>
        <property name="text">
         <string>تحديث</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>280</y>
          <width>81</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(78, 166, 189);
color: rgb(255, 0, 0);
</string>
        </property>
        <property name="text">
         <string>حذف</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="stock">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>230</y>
          <width>42</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;ManaraDocs Amatti Font&quot;;
background-color: rgb(202, 202, 202);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>تحديث</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>//dir2/d/620-1-en-9f03157684be7d738e3c2fed29f12326.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
